packages/next-tinacms-cloudinary/src/handlers.ts

```diff
@@ -122,34 +123,37 @@ async function listMedia(
     }
     let folders: string[] = []
     let folderRes = null
+    const loadFolders = !(filesOnly == 'true')
```

Comment:
Should we be using `==` instead of `===`

Reply:
Good idea